### ISSUE TITLE
Provide an option to disable menus

### ISSUE THREAD

**Issue by joj** [madskristensen/DialToolsForVS#4]
Also not an issue but a suggestion:
I never use some of the features (the Editor menu, for instance) and I would love a page in Tools -> Options to enable/disable menus.

I can contribute if this idea is accepted :)

**Comment by madskristensen** [comment]
Yes, that would be helpful. Since each provider is MEF'ed in, it would be ideal if the options page was dynamic so it could list all the MEF'ed in providers instead of a hard coded list. However, it's soooo much easier to do the hard coded list and since there are no other extension for this extension, it should be fine doing hard coding.

**Comment by joj** [comment]
I'll try it out and submit a PR :)

**Comment by bjorndaniel** [comment]
@joj Did you ever get around to doing this? If not, I have a fork of the repo with a version of this I could do a PR for if you don't mind?

**Comment by madskristensen** [comment]
I didn't do it, so a PR would be great!
